src/rgw/motr/gc/gc.h

```diff
@@ -64,19 +64,20 @@ struct motr_gc_obj_info {
   std::string tag;             // gc obj unique identifier
   std::string name;            // fully qualified object name
   Meta mobj;                   // motr obj
-  std::time_t time;            // deletion time
+  std::time_t deletion_time;   // time when object got deleted
```

Comment:
suggestion :
time when object got deleted  -> time when Motr object requested to delete

Reply:
Done